packages/synchro-charts/src/utils/dataConstants.ts

```diff
@@ -25,3 +27,53 @@ export enum ChartType {
   BarChart = 'bar-chart',
   LineChart = 'line-chart',
 }
+
+export const DIAL_SIZE_CONFIG = {
```

Comment:
we want to do this, but in `IoT App Kit`

Reply:
It will have DIAL_SIZE_CONFIG in `IoT App Kit`. This is for test in synchro-charts.  Or Just input specific size like `fontSize: 96px; dialThickness: 50px`, not define DIAL_SIZE_CONFIG when cypress test in synchro-charts?

Reply:
we'll move these tests into iot app kit instead. In synchro charts we will still want some visual snapshots that show the overall settings work

Reply:
Done --- Move to the cypress test `dial.spec.ts` file